app/models/preserved_objects_primary_moab.rb

```diff
@@ -5,4 +5,10 @@
 class PreservedObjectsPrimaryMoab < ApplicationRecord
   belongs_to :preserved_object, inverse_of: :preserved_objects_primary_moab
   belongs_to :complete_moab, inverse_of: :preserved_objects_primary_moab
+
+  validate :complete_moab_matches_preserved_object
+
+  def complete_moab_matches_preserved_object
+    raise "Complete Moab #{complete_moab.preserved_object_id} does not match" unless complete_moab.preserved_object == preserved_object
```

Comment:
This currently raises a `RuntimeError` open to suggestions about more specific error that should be raised instead.

Reply:
I haven't read the issue linked to this PR, but I wonder if we want to raise an exception here or populate the validation errors list, e.g.:

```ruby
def complete_moab_matches_preserved_object
  if complete_moab.preserved_object != preserved_object
    errors.add(:preserved_object, 'must match the preserved object associated with the complete moab')
  end
end
```

I might be surprised by a validation raising an error instead of conforming to the validation interface.

Reply:
Thanks @mjgiarlo, my first implementation was using the validation error list so I can easily revert back.

Reply:
haha! It happens. :smiley:  Thanks, @jermnelson